build.gradle

```diff
@@ -18,6 +18,7 @@ buildscript {
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.0'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-android-extensions:$kotlin_version"
```

Comment:
Can you tell me why you added this? 
For the Kotlin Android Extension, isn't it necessary?

Reply:
Kotlin Android Extension is used on next code.
~~~Kotlin
    class ViewHolder(parent: ViewGroup, albumSize: Int) : RecyclerView.ViewHolder(
            LayoutInflater.from(parent.context).inflate(R.layout.album_item, parent, false)) {
        val imgALbumThumb = itemView.img_album_thumb
        val txtAlbumName = itemView.txt_album_name
        val txtAlbumCount = itemView.txt_album_count

        init {
            imgALbumThumb.layoutParams = LinearLayout.LayoutParams(albumSize, albumSize)
        }
    }
~~~

itemView.txt_album_name is the Android extension feature.
Of course, these can be replaced findViewById.

Reply:
I know 'kotlin-android-extensions' can use it without having to declare this. 

I know I only need `kotlin-gradle-plugin`.

Reply:
I understand it.
I removed and verified.